core/common/src/ElementProps.ts

```diff
@@ -251,16 +250,16 @@ export interface GeometricElement2dProps extends GeometricElementProps {
   typeDefinition?: RelatedElementProps;
 }
 
-/** JSON representation of a [TextAnnotation2d]($backend).
+/** Properties that define a [TextAnnotation2d]($backend).
  * @public @preview
  * @extensions
  */
 export interface TextAnnotation2dProps extends GeometricElement2dProps {
-  jsonProperties?: {
```

Comment:
isnt removing this breaking?

Reply:
`jsonProperties` is defined on `ElementProps`; this redeclaration just embellished the type.